pallets/randomness/src/lib.rs

```diff
@@ -225,6 +225,12 @@ pub mod pallet {
 	pub(crate) type RandomnessResults<T: Config> =
 		StorageMap<_, Twox64Concat, RequestType<T>, RandomnessResult<T::Hash>>;
 
+	/// Previous local per-block VRF randomness
+	/// Set in `on_finalize` of last block
+	#[pallet::storage]
+	#[pallet::getter(fn previous_local_vrf_output)]
+	pub type PreviousLocalVrfOutput<T: Config> = StorageValue<_, Option<T::Hash>, ValueQuery>;
```

Comment:
```suggestion
	pub type PreviousLocalVrfOutput<T: Config> = StorageValue<_, T::Hash, ValueQuery>;
```

Reply:
`PreviousLocalVrfOutput` don't need to be an Option. By default (at genesis or first block after upgrade) this will be the default hash